node_modules/oae-content/lib/internal/util.js

```diff
@@ -104,10 +106,35 @@ var augmentContent = module.exports.augmentContent = function(ctx, content, dura
             content.previews.wideUrl = getSignedDownloadUrl(ctx, content.previews.wideUri, duration, offset);
             delete content.previews.wideUri;
         }
+
+        if (content.previews.video) {
+            content.previews.video = _getSignedVideoUrl(ctx, content.previews.key);
+        }
     }
 };
 
 /**
+ * Generate a signed url that can be used to show video previews in the browser for a video file stored in AWS S3
+ *
+ * @param  {Context}    ctx         Standard context object containing the current user and the current tenant
```

Comment:
FYI, more up-to-date wording on this parameter description is just "Current request context". Less to type, less to read, less redundant, and less half truths :P